rosidl_typesupport_introspection_c/CMakeLists.txt

```diff
@@ -15,7 +15,8 @@ find_package(ament_cmake_python REQUIRED)
 find_package(ament_cmake_ros REQUIRED)
 
 ament_export_dependencies(rosidl_cmake)
-ament_export_dependencies(rosidl_generator_c)
+# ament_export_dependencies(rosidl_generator_c)
```

Comment:
Please remove this line rather than commenting it out.

Same below.

Reply:
Removed comments